db/schema.rb

```diff
@@ -48,6 +48,7 @@
   create_table "parties", force: :cascade do |t|
     t.string "name"
     t.string "color"
+    t.string "shortName"
```

Comment:
I **think** this is the right thing to do? I don't know anything about how Ruby does databases.

Reply:
It is but I guess we need to add a migration, and also for consistency we should stick to the `snake_case` convention rather than `camelCase`.